src/org/openbravo/erpCommon/ad_forms/AcctServer.java

```diff
@@ -129,6 +133,7 @@ public abstract class AcctServer {
   public String M_Warehouse_ID = "";
   public String Posted = "";
   public String DocumentType = "";
+  private String hasDocumentType = "";
```

Comment:
then the another vars are wrong? because it are as private.

Reply:
Yes, all the other public constants seem to be wrong. SonarQube raises issues for all of them.

I could change them all to `private` for the code to look less confusing

Reply:
Discussed with @Matias-Bernal. The variable will be put as public, ignoring the sonar issue. This will be done for the new code to be consistent with the older one